Sdk/Exceptions/AllException.cs

```diff
@@ -48,7 +48,7 @@ public override string Message
                     return string.Format(CultureInfo.CurrentCulture,
                                          "{0}Item: {1}{2}{3}",
                                          indexString,
-                                         error.Item2 == null ? "null" : error.Item2.ToString().Replace(Environment.NewLine, spaces),
+                                         error.Item2?.ToString().Replace(Environment.NewLine, spaces),
```

Comment:
I think you need `.Item2?.ToString()?.Replace` because calling .Replace on null will null-ref.